src/store/getters.ts

```diff
@@ -23,7 +28,7 @@ export default {
 
   /** Is True if entity is a Limited Company. */
   isLtd (state): boolean {
-    return (state.entityType === CorpTypeCd.LIMITED_CO)
+    return (state.entityType === CorpTypeCd.BC_COMPANY)
```

Comment:
This is a change from LLC to BC. Intended?

Reply:
How about renaming the getter?

Reply:
An LTD is 'BC' in Lear, so i think this is correct. 
In our corpTypes 'BC' was labeled as BC_COMPANY though, is that still accurate?

Reply:
Maybe check corp types with a BA.

Please rename getter (eg, isBcCompany).

Reply:
Kk, will update!